application/views/js_init/phonebook/js_group.php

```diff
@@ -94,44 +94,37 @@
 			$.post("<?php echo site_url('messages/compose_process') ?>", $("#composeForm").serialize(), function(data) {
 				$("#compose_sms_container").html(data);
 				$("#compose_sms_container" ).dialog( "option", "buttons", { "Okay": function() { $(this).dialog("destroy"); } } );
-				setTimeout(function() {$("#compose_sms_container").dialog('destroy')} , 1500);
+				setTimeout(function() {if ($("#compose_sms_container").hasClass('ui-dialog-content')) { $("#compose_sms_container").dialog('destroy')}} , 1500);
 			});
 			}
 		},
 		'<?php echo lang('kalkun_cancel')?>': function() { $(this).dialog('destroy');}
 	    }
 	  });
-	});
 	$("#compose_sms_container").dialog('open');
+	});
 	return false;
 });
 	
 // select all
-$("a.select_all").click(select_all = function(){
-$(".select_group").attr('checked', true);
+$("a.select_all").on("click", select_all = function(){
+$(".select_group").prop('checked', true);
 $(".contact_list").addClass("messagelist_hover");
 return false;
 });
 
 // clear all
-$("a.clear_all").click(clear_all = function(){
-$(".select_group").attr('checked', false);
+$("a.clear_all").on("click", clear_all = function(){
+$(".select_group").prop('checked', false);
 $(".contact_list").removeClass("messagelist_hover");
 return false;
 }); 
 
 // input checkbox
-$("input.select_group").click(function(){
-if($(this).attr('checked')==true) $(this).parents('div:eq(2)').addClass("messagelist_hover");
+$("input.select_group").on("click", function(){
+if($(this).prop('checked')==true) $(this).parents('div:eq(2)').addClass("messagelist_hover");
 else $(this).parents('div:eq(2)').removeClass("messagelist_hover");
 });
 
-// Show menu on hover
-$("tr").hover(function() {
-	$(this).find("span.pbk_menu").show();
-},function() {
- 	$(this).find("span.pbk_menu").hide();
-});  
-
```

Comment:
have you looked at the commit message? ;)

Reply:
> Simply remove its use. It brought mainly distraction

Reply:
Also, when on a phone, you simply can't "hover", and the command would not be available at all.

Reply:
Today Kalkun isn't responsive at all. Maybe we should make kalkun responsive to be used from both phone/desktop. 

Having certain actions on hover makes the page clean. This was inspired by how google contacts any similar apps have actions on hover.
![image](https://user-images.githubusercontent.com/709368/145224801-5183606a-268d-433d-9322-5569edd1bf49.png)
![image](https://user-images.githubusercontent.com/709368/145224989-0d83b920-6562-4ff4-b6d3-23d7dd08da56.png)

Reply:
That's expected behaviour with that change.

When you arrive on the screen you know directly what actions you can do. If hidden by default and only enabled on hover (as it used to be) you may discover them somehow by chance.

I really prefer without hover. Now if you insist I can make it like before, but on touchscreen there will be no access to that function.

I don't think this makes it unresponsive, on the contrary, it remove complexity.

Reply:
I think this can be solved in a better way.  Ideally, the UI & behaviour should be different on touch and non-touch devices. But given that the current kalkun UI doesn't support that, I think I may have a way to bring back the functionality only on devices that isn't touch. Will raise a separate pr for that.